barretenberg/cpp/src/barretenberg/crypto/schnorr/c_bind.hpp

```diff
@@ -3,11 +3,11 @@
 #include "barretenberg/ecc/curves/grumpkin/grumpkin.hpp"
 #include "multisig.hpp"
 
-extern "C" {
```

Comment:
what about the rest of the c_bind.hpps? Seems like there are plenty that still have the extern "C"?

Reply:
scrubbed